src/dashboard/luasrc/view/ids-priv/config.htm

```diff
@@ -131,14 +131,15 @@
 					return_string = return_string .. "<td class=\"mac row_" .. tostring(config_count) .. "\">" .. mac .. "</td>" .. "\n"
 					return_string = return_string .. "<td class=\"int row_" .. tostring(config_count) .. "\">" .. interface .. "</td>" .. "\n"
 					return_string = return_string .. "</tr>" .. "\n"
+					config_count = config_count + 1
 				end
 			end
 		end
 		return_string = return_string .. "</table>"
 		return return_string
 	end
 
-	local data = format_ip_neighbors(luci.sys.exec("ip neigh | grep -h \"192.\""))
+	local data = format_ip_neighbors(luci.sys.exec("ip neigh"))
```

Comment:
Removed filter in case you want the ability to add more devices @MartinLPetrauskas

Reply:
Thank you kind sir.

Reply:
What is this system call doing?

Reply:
https://man7.org/linux/man-pages/man8/ip-neighbour.8.html

Reply:
The `arp -a` command most commonly used in this instance isn't actually a callable shell command, so I had to use this as an alternative.